python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1693,49 +1694,67 @@ def create_run(
 
         return dagster_run
 
-    def _get_skipped_entity_keys(
-        self, run_id: str
+    def _get_keys_to_reexecute(
+        self, run_id: str, execution_plan_snapshot: "ExecutionPlanSnapshot"
     ) -> tuple[AbstractSet["AssetKey"], AbstractSet["AssetCheckKey"]]:
         """For a given run_id, return the set of asset keys and asset check keys that were planned but
```

Comment:
Nitty; but I want this docstring to be a "de facto" comment explaining the exact mechanism for when each asset / asset check will or will not be retried. Just want it to be more exhaustive I guess.